README.org

```diff
@@ -326,6 +326,83 @@ namespace-keyword := {:general | :talk | :user | :user-talk | :wikipedia | :wiki
        [:language string] ; like en-US
        [:time-range {:day | :week | :month | :year}])
 #+end_src
+** Startpage
+All the settings which are accessible from the search page can be configured.
+In order to specify settings from Startpage's "Settings" page, set `:settings-string` to the
+hexadecimal number situated after "prfe=" in the URL displayed in the "Save without cookie" section.
+Syntax:
+#+begin_src
+    (startpage [:shortcut string]
+               [:fallback-url quri:uri]
+               [:base-search-url string]
+               [:object {:web | :images | :videos | :news }
+               ;; If `language-ui` or `language-results` are set in Startpage's
+               ;; settings page, either via a cookie or the settings string, the
+               ;; settings below will be overridden.
+               [:language-ui {:dansk | :deutsch | :english | :espanol | :francais |
+                              :nederlands | :norsk | :polski | :portugues | :svenska}]
+               [:language-results  {:afrikaans | :albanian | :amharic | :arabic |
+                                    :azerbaijani | :basque | :belarusian | :bengali |
+                                    :bihari | :bosnian | :bulgarian | :catalan |
+                                    :croatian | :czech | :dansk | :deutsch | :english |
+                                    :english_uk | :espanol | :esperanto | :estonian |
+                                    :fantizhengwen | :faroese | :francais | :frisian |
+                                    :gaelic | :galician | :georgian | :greek |
+                                    :gujarati | :hangul | :hebrew | :hindi | :hungarian |
+                                    :icelandic | :indonesian | :interlingua |
+                                    :irish | :italiano | :javanese | :jiantizhongwen |
+                                    :kannada | :latin | :latvian | :lithuanian |
+                                    :macedonian | :malay | :malayalam | :maltese |
+                                    :marathi | :nederlands | :nepali | :nihongo |
+                                    :norsk | :occitan | :persian | :polski | :portugues |
+                                    :punjabi | :romanian | :russian | :serbian |
+                                    :sinhalese | :slovak | :slovenian | :sudanese |
+                                    :suomi | :svenska | :swahili | :tagalog |
+                                    :tamil | :telugu | :thai | :tigrinya |
+                                    :turkce | :ukrainian | :urdu | :uzbek |
+                                    :vietnamese | :welsh | :xhosa | :zulu}]
+               ;; web search arguments:
```

Comment:
Good point, although most actually do need a prefix because they only apply to web searches.
- `with_date` only applies to web and news, so I think we should probably keep things as they are
- The "family filter" seems to apply to all but news, so I'll remove the prefix.
- `region` only seems to apply to web searches